examples/devices/lighty-network-topology-multiple-devices/src/main/java/io/lighty/netconf/device/topology/Main.java

```diff
@@ -0,0 +1,194 @@
+        LOG.info("/    |    \\  ___/|  | |    `   \\  ___/\\   /|  \\  \\__\\  ___/");
+        LOG.info("\\____|__  /\\___  >__|/_______  /\\___  >\\_/ |__|\\___  >___  >");
+        LOG.info(" \\/     \\/            \\/     \\/             \\/    \\/");
+        LOG.info("[https://lighty.io]");
+
+        //1. Load models from classpath
+        Set<YangModuleInfo> modules = ModelUtils.getModelsFromClasspath(
+                ModuleId.from("urn:tech.pantheon.netconfdevice.network.topology.rpcs",
+                        "network-topology-rpcs",
+                        "2018-03-20"),
+                ModuleId.from("urn:opendaylight:netconf-node-topology",
+                        "netconf-node-topology",
+                        "2015-01-14"));
+
+        //2. Initialize DataStores
+        InputStream initialOperationalData = null;
+        InputStream initialConfigurationData = null;
+        if (initDatastore) {
+            initialOperationalData =
+                    Main.class.getResourceAsStream("/initial-network-topo-operational-datastore.xml");
+            initialConfigurationData =
+                    Main.class.getResourceAsStream("/initial-network-topo-config-datastore.xml");
+        }
+
+        //3. Initialize RPCs
+        NetworkTopologyServiceImpl networkTopologyService = new NetworkTopologyServiceImpl();
+
+        //4. Initialize Netconf device
+        NetconfDevice netconfDevice = new NetconfDeviceBuilder()
+                .setCredentials("admin", "admin")
+                .setBindingPort(params.startingPort)
+                .withModels(modules)
+                .withDefaultRequestProcessors()
+                .withDefaultCapabilities()
+                .setThreadPoolSize(params.threadPoolSize)
+                .setDeviceCount(params.deviceCount)
+                .setInitialOperationalData(initialOperationalData)
+                .setInitialConfigurationData(initialConfigurationData)
+                .withRequestProcessor(new NetworkTopologyServiceGetTopologiesProcessor(networkTopologyService))
+                .withRequestProcessor(new NetworkTopologyServiceGetTopologyByIdProcessor(networkTopologyService))
+                .withRequestProcessor(new NetworkTopologyServiceGetTopologyIdsProcessor(networkTopologyService))
+                .withRequestProcessor(new NetworkTopologyServiceGetNodeFromTopologyProcessor(networkTopologyService))
+                .withRequestProcessor(new NetworkTopologyServiceCreateTopologyProcessor(networkTopologyService))
+                .withRequestProcessor(new NetworkTopologyServiceAddNodeToTopologyProcessor(networkTopologyService))
+                .withRequestProcessor(new NetworkTopologyServiceRemoveTopologyProcessor(networkTopologyService))
+                .withRequestProcessor(new NetworkTopologyServiceRemoveAllTopologiesProcessor(networkTopologyService))
+                .withRequestProcessor(new NetworkTopologyServiceRemoveNodeProcessor(networkTopologyService))
```

Comment:
To demonstrate multiple devices I believe we do not need so many processors
In future it will be easier to maintain less complex example, still we want to show here how to create multiple-devices simulator.
Please change this example to be simple example just for multiple-devices demonstration

Reply:
e.g. you can create simple YANG model with one container and one rpc... that is enough